mac

```diff
@@ -237,9 +237,10 @@ fancy_echo "Visual Studio Code configured."
 fancy_echo "Configuring nodenv..."
 eval "nodenv init -"
 
-# nodenv patch
+# add nodenv initialization to the shell file, if not already present
 fancy_echo "patching nodenv init into .zshrc file"
-echo "eval \"\$(nodenv init -)\"" >> .zshrc
+nodenv_init="eval \"\$(nodenv init -)\""
+cat "$HOME/.zshrc" | grep "$nodenv_init" > /dev/null || echo "$nodenv_init" >> "$HOME/.zshrc"
```

Comment:
we append the init script only if there's no already

Reply:
you're right, much better.. and also using $HOME allows us to run the script wherever we want!